bokeh/document.py

```diff
@@ -752,10 +752,6 @@ def validate(self):
             None
 
         '''
-        # logging.basicConfig is a no-op if there's already
-        # some logging configured. We want to make sure warnings
-        # go somewhere so configure here if nobody has.
-        logging.basicConfig(level=logging.INFO)
```

Comment:
My impression of the issue was that you intended for a different custom logger to be configured in `Document`, not that no logger at all be configured. These are validation errors, they explain usage problems that might otherwise result in e.g. a blank graph with no other explanation. We put this here because there are warnings and errors that must be relayed, unconditionally, or else our support burden increases beyond maintainable levels.